### ISSUE TITLE
result

### ISSUE THREAD

**Issue by Zhudogsi** [mdswyz/DMD#13]
Hello, may I ask if the accuracy result in this paper is a good thing of has0 and no-has0? What's the difference between aligned and misaligned results? What method is used to align different modes? thank you for your reply.

**Comment by xueyuG** [comment]
I'm also trying to reproduce the model. Judging from the code provided by the author, they only calculate non0_acc2 but not has0_acc2.

**Comment by hustcselwb** [comment]
请问论文中的结果是同一个epoch下的结果吗，还是不同epoch下的最优结果。

**Comment by Raymond-Qiancx** [comment]
> I'm also trying to reproduce the model. Judging from the code provided by the author, they only calculate non0_acc2 but not has0_acc2.

Yes, i have the same question, and my running results is 2% lower than the paper illustrated.

**Comment by Issac304** [comment]
确实低了一点，解决了吗？

**Comment by tbzzzzz** [comment]
May I ask if there is a problem with my settings, as the unaligned version of the paper gave 83.5, but I tested and found it to be almost 3%?
